src/sentry/tasks/post_process.py

```diff
@@ -152,7 +151,7 @@ def index_event_tags(organization_id, project_id, event_id, tags, group_id=None,
         try:
             # handle replaying of this task
             with transaction.atomic():
```

Comment:
While I'm here, I don't understand the point of `atomic` here. There's only one statement in here, it's already atomic...?

Reply:
It's to handle the `IntegrityError` otherwise it'll break the whole transaction.

Reply:
So here, I'd argue that we bring this transaction within `create_event_tag` since now it's a bit of an implementation detail leaking out.

Reply:
Ya, now that I get why it's there I agree.